src/elements/content-picker/ContentPicker.js

```diff
@@ -820,6 +827,55 @@ class ContentPicker extends Component<Props, State> {
         });
     };
 
+    /**
+     * Show the inline shared link dropdown
+     * @param {BoxItem} item - The item (folder, file, weblink)
+     * @returns {Promise}
+     */
+    showSharedLinkDropdown = (item: BoxItem): void => {
+        const { id, type }: BoxItem = item;
+
+        switch (type) {
+            case TYPE_FOLDER:
+                this.api.getFolderAPI().getFolderFields(id, this.handleSharedLinkResponse, () => {}, {
```

Comment:
Is swallowing the error response desirable? If so, can we use `lodash/noop`, instead?